### PR TITLE
OPENEUROPA-657: Rename the module from eu_login to oe_auth.

### PR DESCRIPTION
**Pull request by drupol**
### Description

Rename the module from eu_login to oe_auth.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# EU Login
+# OpenEuropa Auth
 
-[![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/eu_login/status.svg?branch=master)](https://drone.fpfis.eu/openeuropa/eu_login)
+[![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/oe_auth/status.svg?branch=master)](https://drone.fpfis.eu/openeuropa/oe_auth)
 
-The EU Login module allows to authenticate against the EU Login service.
+The OE Auth module allows to authenticate against the European Commission login service.
```
Comment (ademarco) [inline README.md]: Shall we maybe use the full name every time we refer to the module? Like `OpenEuropa Auth`

File: README.md [hunk 2]
```diff
@@ -15,14 +15,14 @@ The EU Login module allows to authenticate against the EU Login service.
...
-The recommended way of installing the EU Login module is via a [Composer-based workflow][2].
+The recommended way of installing the OE Auth module is via a [Composer-based workflow][2].
 
 In your Drupal project's main `composer.json` add the following dependency:
 
 ```json
 {
     "require": {
-        "openeuropa/eu_login": "dev-master"
+        "openeuropa/oe_auth": "dev-master"
```
Comment (ademarco) [inline README.md]: Let's discourage people from using dev-master, I suggest the following change:

```
"openeuropa/oe_auth": "^0.1"
```

Comment (netlooker) [reply README.md]: Nice spot. I've missed that one.

**Comment by ademarco** [comment]
@drupol we should also remove the `frpc-tunnel:` section in `.drone.yml` as we are not using it for the time being.

**Comment by drupol** [comment]
I pushed a commit to update the .drone.yml file.

**Pull request merged**